### PR TITLE
Update QuickPerf projects

### PR DESCRIPTION
**Pull request by jeanbisutti**
Hello,

I have updated the QuickPerf projects. I am the main maintainer of these projects.

Jean

### CODE REVIEW

**Review by ritwik12 — CHANGES_REQUESTED** [review]
@jeanbisutti I guess you forgot to change label names on those repos. There it is still Thumbup good first issue

**Comment by jeanbisutti** [comment]
> @jeanbisutti I guess you forgot to change label names on those repos. There it is still Thumbup good first issue

@ritwik12 , thanks for your review. I am sorry but [I sew only `name: good first issue`](https://github.com/up-for-grabs/up-for-grabs.net/pull/2054/files), as it is done in other projects

**Comment by ritwik12** [comment]
@shiftbot

**Comment by ritwik12** [comment]
@jeanbisutti  please update those in your repos.

**Comment by shiftkey** [comment]
It is currently failing because the label name on these specific projects starts with the emoji character `:thumbsup:` for both projects:



If you look at the API response, it'll show the same text:



We don't do any fuzzy matching here, because we need to lookup through the API which requires an exact match for the label name.

**Comment by ritwik12** [comment]
@shiftkey Do we have a command to rebuild @shiftbot for changes such as these. Because now there are no commits here but only name change on the owners repo.

**Comment by shiftkey** [comment]
@ritwik12 validating the entire file would happen on any field change, and I still think it'll complain because of the label changes in this PR (this would be fine if those are reverted). 

Re-running the validation isn't supported yet, open an issue over on https://github.com/shiftkey/webhooks if you'd like to chat about that further.

**Comment by ritwik12** [comment]
Cool. Thanks @shiftkey

File: _data/projects/maven-test-bench.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ tags:
 - performance
 - quickperf
 upforgrabs:
-  name: ":thumbsup: good first issue"
+  name: good first issue
```
Comment (shiftkey) [inline _data/projects/maven-test-bench.yml]: Reverting this to it's old value is what's required here:

```suggestion
  name: ":thumbsup: good first issue"
```

File: _data/projects/quickperf.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ tags:
 - performance
 - quickperf
 upforgrabs:
-  name: ":thumbsup: good first issue"
+  name: good first issue
```
Comment (shiftkey) [inline _data/projects/quickperf.yml]: Reverting this to it's old value is what's required here:

```suggestion
  name: ":thumbsup: good first issue"
```

File: _data/projects/maven-test-bench.yml (hunk 1 above)
Comment (jeanbisutti) [reply _data/projects/maven-test-bench.yml]: Thanks @shiftkey

Comment (ritwik12) [inline _data/projects/maven-test-bench.yml]: @jeanbisutti Here, also you will have to revert.
```suggestion
  name: : "thumbsup: good first issue"
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@jeanbisutti Sorry my suggestion had an extra : 

please make it ":thumbsup: good first issue" from : ":thumbsup: good first issue"

File: _data/projects/maven-test-bench.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ tags:
 - performance
 - quickperf
 upforgrabs:
-  name: ":thumbsup: good first issue"
+  name: : "thumbsup: good first issue"
```
Comment (ritwik12) [inline _data/projects/maven-test-bench.yml]: @jeanbisutti Please remove this extra colon outside ""
```suggestion
  name: ":thumbsup: good first issue"
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@jeanbisutti Thanks a lot!!

**Comment by jeanbisutti** [comment]
Huge thanks for your help @ritwik12!

**Pull request merged**